aviary/mission/flops_based/ode/mission_ODE.py

```diff
@@ -198,11 +217,43 @@ def setup(self):
                            ],
                            promotes_outputs=['initial_mass_residual'])
 
+        if analysis_scheme is AnalysisScheme.SHOOTING:
+            SGM_required_outputs = {
+                Dynamic.Mission.ALTITUDE_RATE: {'units': 'm/s'},
+            }
+            add_SGM_required_outputs(self, SGM_required_outputs)
+
+        print_level = 0 if analysis_scheme is AnalysisScheme.SHOOTING else 2
```

Comment:
Kind of wonder why the default is 2 for colocation.

Reply:
that's what it was on [line 208](https://github.com/OpenMDAO/Aviary/pull/97/files#diff-d6e2c364a0164fec8fd278a17c15194b88f0b576c84ba74607669322957660d6L208), so I didn't want to change it.